packages/legend-data-cube/src/stores/core/DataCubeQueryEngine.ts

```diff
@@ -74,6 +74,10 @@ export enum DataCubeFunction {
   // PERCENTILE = 'meta::pure::functions::math::percentile',
 }
 
+export enum DataCubeEngineFunction {
```

Comment:
we use `DataCubeFunction` instead, if you need extract the function name from `DataCubeFunction`, use `DataCubeQueryBuilderUtils._functionName()`, if you need to match a string to a function name, use `matchFunctionName()`

Reply:
I removed this, but I still use 2 enums which I am not sure is the right thing to do

Reply:
removed this completely